lib/tasks/data.rake

```diff
@@ -0,0 +1,65 @@
+namespace :data do
+  require 'open-uri'
+  require 'json'
+
+  def generate_shortname(name)
+    known_exceptions = {
+      "Wellington (A)" => "wellington_nsw",
+      "Kingston (DC)" => "kingston_sa",
+      "Latrobe (M)" => "latrobe_tas",
+      "Murray (A)" => "murray_nsw",
+      "Flinders (S)" => "flinders_qld",
+      "Central Highlands (M)" => "central_highlands_tas"
+    }
+
+    known_exceptions[name] || name.split(" (").first.downcase.gsub(" ", "_")
+  end
+
+  path = File.dirname(__FILE__) + "/../../tmp/lga_2015_aust.geojson"
```

Comment:
This isn't a big file, better to just fetch it each time and hold it in memory? If that's not appropriate maybe a Tempfile would be better.